src/app/rpmostree-builtin-status.c

```diff
@@ -307,33 +314,34 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
             {
             case AUTO_UPDATE_SDSTATE_TIMER_UNKNOWN:
               {
-                g_print ("(%s; unknown timer state)\n", policy);
+                g_print ("unknown state: %s\n", RPMOSTREE_AUTOMATIC_TIMER_UNIT);
                 break;
               }
             case AUTO_UPDATE_SDSTATE_TIMER_INACTIVE:
               {
-                g_print ("(%s; timer inactive)\n", policy);
+                g_print ("%s inactive\n", RPMOSTREE_AUTOMATIC_TIMER_UNIT);
```

Comment:
More bikeshed: I like the with `%s:` prefix going on down below. Maybe let's just use that here and higher up too to be consistent? I.e.:

```
%s: unknown state
%s: inactive
```

?